src/org/usfirst/frc2175/config/GamepadConfig.java

```diff
@@ -30,10 +32,16 @@ public String getPropertyFileName() {
     protected void configure(Properties properties) {
         gamepad = new Joystick(getIntPropertyValue("gamepad.port", properties));
 
-        extendCatapult = new JoystickButton(gamepad, getIntPropertyValue("button.catapult.extend", properties));
-        retractCatapult = new JoystickButton(gamepad, getIntPropertyValue("button.catapult.retract", properties));
-
-        testAction = new JoystickButton(gamepad, getIntPropertyValue("button.action.test", properties));
+        extendCatapult = new JoystickButton(gamepad,
+                getIntPropertyValue("button.catapult.extend", properties));
+        retractCatapult = new JoystickButton(gamepad,
+                getIntPropertyValue("button.catapult.retract", properties));
+        runIntakeIn = new JoystickButton(gamepad,
+                getIntPropertyValue("button.catapult.intake.in", properties));
+        runIntakeOut = new JoystickButton(gamepad,
+                getIntPropertyValue("button.catapult.intake.out", properties));
```

Comment:
The intake is not really part of the catapult, so a better property name might just be `button.intake.out`. Same for `.in` above. And don't forget to change this in gamepad.properties.